frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -23,6 +23,14 @@ export const getSelectOverrides = (): MantineThemeOverride["components"] => ({
     styles: (theme, _, { size = "md" }) => ({
       ...getSelectInputOverrides(theme, size),
       ...getSelectItemsOverrides(theme, size),
+      // For epic (metabase#38699)
```

Comment:
Fix this problem I posted in a [Slack thread](https://metaboat.slack.com/archives/C057WD5L0JG/p1707929338121809)